globalConfig.json

```diff
@@ -31,686 +31,749 @@
+                    "pattern": "^https:\\/{2}(?:\\S)+\\.\\w{2,}$",
+                    "errorMsg": "URL must begin with https:// and end with top-level domain. E.g.: https://app.us1.dataset.com"
                   }
                 ]
               },
-              "defaultValue": "http",
-              "field": "proxy_type"
+              {
+                "field": "tenant",
+                "label": "Tenant",
+                "type": "text",
+                "help": "Tenant field can be True/False/Blank. If True, entire Tenant will be considered or if False, only the Account IDs will be considered for queries. If Blank neither consider",
+                "required": false,
+                "defaultValue": "",
+                "validators": [
+                    {
+                        "type": "regex",
+                        "pattern": "^(true|True|TRUE|false|False|FALSE)$",
+                        "errorMsg": "Tenant must be a value of True or False or empty"
+                    }
+                ]
             },
             {
-              "field": "proxy_url",
-              "label": "Host",
-              "type": "text",
-              "validators": [
-                {
-                  "type": "string",
-                  "minLength": 0,
-                  "maxLength": 4096,
-                  "errorMsg": "Max host length is 4096"
-                }
-              ]
+                "field": "account_ids",
+                "label": "Account IDs",
+                "type": "text",
+                "help": "Provide comma separated values of Account IDs on which the queries should run, when Tenant is set to False.",
+                "required": false,
+                "defaultValue": "",
+                "validators": [
+                    {
+                        "type": "regex",
+                        "pattern": "^\\d+(,\\d+)*$",
+                        "errorMsg": "Account IDs must be a comma separated ids"
+                    }
+                ]
             },
             {
-              "field": "proxy_port",
-              "label": "Port",
-              "type": "text",
-              "validators": [
-                {
-                  "type": "number",
-                  "range": [
-                    1,
-                    65535
-                  ]
-                }
-              ]
+                "field": "authn_token_part_one",
+                "label": "AuthN API Token First Part",
+                "type": "text",
+                "help": "Encrypted part of the Token required to enable inputs and SPL command.",
+                "required": false,
+                "defaultValue": "",
+                "encrypted": true,
+                "validators": [
+                    {
+                        "type": "string",
+                        "minLength": 0,
+                        "maxLength": 220,
+                        "errorMsg": "Max length of password is 220"
```

Comment:
Does the `password` is the right word here?

Reply:
yes, followed the same naming conversation as followed by the previous config fields